src/etc/inc/notices.inc

```diff
@@ -461,10 +486,67 @@ function curl_post_telegram($url) {
 	return $curl_post_telegram_data;	//json encoded
 }
 
+/****f* notices/notify_via_pushover
+ * NAME
+ *   notify_via_pushover
+ * INPUTS
+ *	 notification string to send to Pushover via API
+ * RESULT
+ *   returns NULL if message was sent
+ ******/
+
+function notify_via_pushover($message, $force = false) {
+	global $config;
+	if ((!isset($config['notifications']['pushover']['enabled']) && (!$force))
+	    || !$config['notifications']['pushover']['apikey']
+	    || !$config['notifications']['pushover']['userkey']) {
+		if ($force) {
+			return gettext("Unable to test Pushover notification without both API Key & User Key set");
+		}
+		return;
+	}
+
+	$url = "https://api.pushover.net/1/messages.json";
+	$data = array(
+		"token" => $config['notifications']['pushover']['apikey'],
+		"user" => $config['notifications']['pushover']['userkey'],
+		"sound" => $config['notifications']['pushover']['sound'],
+		"priority" => $config['notifications']['pushover']['priority'],
+		"retry" => $config['notifications']['pushover']['retry'],
+		"expire" => $config['notifications']['pushover']['expire'],
+		"message" => "{$config['system']['hostname']}.{$config['system']['domain']}\n{$message}"
+	);
+	$result = json_decode(curl_post_pushover($url, $data), true);
+	if (is_array($result)) {
+		if ($result['status']) {
+			unset($err_msg);
+		} else {
+			$err_msg = sprintf(gettext("Failed to send Pushover notification. Error received was: {$result['errors']['0']}"));
```

Comment:
You must use `%s` on gettext parameter and the pass the variable to sprintf 2nd parameter, otherwise it makes no sense to use sprintf